statsmodels/base/tests/test_generic_methods.py

```diff
@@ -196,7 +198,12 @@ def test_zero_collinear(self):
         # TODO: Can we choose a test case without this issue?
         #  If not, should we be getting this warning for all
         #  model subclasses?
+        # TODO: Investigate how to resolve unseen warnings for Pyodide
+        # Most likely coming from NumPy.linalg + lack of fp exceptions
+        # support under WASM
         warn_cls = HessianInversionWarning if isinstance(mod, sm.GLM) else None
```

Comment:
` warn_cls = HessianInversionWarning if isinstance(mod, sm.GLM) and not is_wasm() else None` and remove indentation

Reply:
Addressed in 1f28d5783f4026742358944f3650fc2d7ecf7f5f.